forum/communications/consumers.py

```diff
@@ -0,0 +1,48 @@
+from channels.generic.websocket import AsyncJsonWebsocketConsumer
+from users.models import CustomUser
+
+
+class ChatConsumer(AsyncJsonWebsocketConsumer):
+    async def connect(self):
+        self.room_name = self.scope["url_route"]["kwargs"]["room_name"]
```

Comment:
Validate if `room_name` is present in the URL route `kwargs` to avoid potential `KeyError`.